Makefile

```diff
@@ -0,0 +1,52 @@
+ESLINT = node_modules/.bin/eslint --config node_modules/sanctuary-style/eslint-es3.json --env es3
```

Comment:
I think for some cases it may be useful to support older versions to allow people to benchmark changes on older versions of V8.